widgetssdk/src/main/res/values/colors.xml

```diff
@@ -12,7 +12,7 @@
     <color name="glia_system_negative_color">#D11149</color>
     <color name="glia_system_agent_bubble_color">#F3F3F3</color>
 
-    <color name="glia_transparent_black_bg">#B3000000</color>
+    <color name="glia_transparent_black_bg">#232323</color>
```

Comment:
No transparency any more?

Should we maybe change the resource name as well or create a new one to avoid confusion?